cc-by-sa/knowledge/modules/using_type/lessons/the_complications_of_typographic_size/content.md

```diff
@@ -84,4 +84,4 @@ Despite some of the limitations and complications mentioned above, there’s sti
 - Be more observant about how different typefaces may require changes in size specifications due to differences in their design or production.
 - Perhaps more important than anything, test the sizing logic on a variety of screens and devices to ensure it works reasonably well in as many environments as possible.
 
-The more knowledgeable and thoughtful designers are to the intricacies of typographics size, the better the experience can be for readers.
\ No newline at end of file
+The more knowledgeable and thoughtful designers are to the intricacies of typographics size, the better the experience can be for readers.
```

Comment:
I believe this shows an empty last line was removed, which is recommended for unix text files, please add back

Reply:
Actually the new version has a new empty line (#88)

Reply:
Perfect then